daprdocs/content/en/reference/cli/dapr-run.md

```diff
@@ -17,19 +17,26 @@ dapr run [flags] [command]
 
 ## Examples
 
```

Comment:
Can we add a dotnet example here, to get dotnet some love! Like this

dapr run --app-id myapp --app-port 5000 -- dotnet run